src/modlistcontextmenu.cpp

```diff
@@ -24,12 +24,14 @@ void ModListGlobalContextMenu::populate(OrganizerCore& core, ModListView* view,
 {
   clear();
 
-  addAction(tr("Install Mod..."), [=]() { view->actions().installMod(); });
 
   auto modIndex = index.data(ModList::IndexRole);
   if (modIndex.isValid() && view->sortColumn() == ModList::COL_PRIORITY) {
     auto info = ModInfo::getByIndex(modIndex.toInt());
     if (!info->isBackup()) {
+
+      addAction(tr("Install mod above..."), [=]() { view->actions().installMod("", index); });
```

Comment:
Why no inside option like for empty mod?

Reply:
Because no one asked for it (yet) :D

Reply:
Done.